hc/settings.py

```diff
@@ -112,7 +112,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'Africa/Nairobi'
```

Comment:
Why have you changed the time zone?

Reply:
I changed it when I was still figuring out how to implement the task we were given, but since we found another alternative of implementing the task I was supposed to revert it to `UTC`.

I will resolve the issue right away.